static/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "short_name": "The Blue Alliance",
+  "name": "The Blue Alliance",
+  "start_url": "/",
+
+  "gcm_sender_id": "103953800507"
```

Comment:
`gcm_sender_id` is the same for all Firebase web clients, it shouldn't matter if this is checked in.